packages/frontend/src/components/Modal/AccountDetailModal.tsx

```diff
@@ -0,0 +1,113 @@
+import Jazzicon from '@metamask/jazzicon'
+import { shortenAddress, useEthers } from '@usedapp/core'
+import copyToClipboard from 'copy-to-clipboard'
+import { useEffect, useRef } from 'react'
+import { CopyIcon, RedirectIcon } from 'src/components//Icons'
+import { Button } from 'src/components/Buttons/Button'
+import { ContentRow, Modal } from 'src/components/Modal/Modal'
+import { useChainId } from 'src/hooks/chainId/useChainId'
+import { Colors } from 'src/styles/colors'
+import { getExplorerAddressLink } from 'src/utils/getExplorerLink'
+import styled, { css } from 'styled-components'
+
+export interface ModalProps {
+  isShown: boolean | undefined
+  onRequestClose: () => void
+  wallet?: string
+}
+
+export const AccountDetailModal = ({ isShown, onRequestClose, wallet = 'Metamask' }: ModalProps) => {
+  const { account, deactivate } = useEthers()
+  const accountIconRef = useRef<any>(null)
+  const chainId = useChainId()
+
+  useEffect(() => {
+    if (account && accountIconRef.current) {
+      accountIconRef.current.innerHTML = ''
+      accountIconRef.current.appendChild(Jazzicon(40, parseInt(account.slice(2, 10), 16)))
+    }
+  }, [account])
+
+  return (
+    <Modal isShown={isShown} onRequestClose={onRequestClose} title="Your account">
```

Comment:
Can we dim the page header as well when modal is open?
Currently:
![image](https://user-images.githubusercontent.com/3901896/168048890-8fbeefdf-7dba-4b6f-808a-f20b8b051fff.png)
Design:
![image](https://user-images.githubusercontent.com/3901896/168049016-c0c80fcb-9338-4c9b-ae2c-75ae16e6d370.png)